packages/help-center/src/components/help-center-content.scss

```
@@ -13,4 +13,21 @@
 			}
 		}
 	}
+
+	// Only show the blender when the footer is present.
+	&:has( .help-center__container-footer ) {
+		.help-center__container-content {
+			&::after {
+				content: '';
+				height: 80px;
```

Maybe this can be a CSS variable that you can read later with JS? That way you don't need to maintain a _magic number_ in JS in the `BLENDER_HEIGHT` constant. 

Didn't test it, but I guess something like this can work:

```suggestion
		.help-center__container-content {
			--blender-height: 80px;
			&::after {
				content: '';
				height: var( --blender-height );
```